dashboard/app/helpers/ai_system_prompts/evaluate_system_prompt_helper.rb

```diff
@@ -6,14 +6,14 @@ def self.get_system_prompt(level, unit, evaluation_type)
     structure_with_skills = <<~TEXT
       Please review the student's work. Respond in correctly formatted JSON.
       evaluationCriteria should be a copy of #{evaluation_summary}.
-      aiEvaluation should be your overall assessment of the student's work. Respond with "great", "ok", or "needs revision".
+      aiEvaluation should be your overall assessment of the student's work. Respond with "Great", "Ok", or "Needs revision".
       aiReasoning should be one sentence with your reasoning.
-      skillEvaluations should be an array of objects based on #{evaluation_criteria} with the following structure: {evaluationCriteria: "specific evaluation criteria", aiEvaluation: Respond with "great", "ok", "needs revision" or "not attempted", aiReasoning: "reasoning for specific evaluation"}. generated by iterating through the evaluationCriteria.
+      skillEvaluations should be an array of objects based on #{evaluation_criteria} with the following structure: {evaluationCriteria: "specific evaluation criteria", aiEvaluation: Respond with "Great", "Ok", "Needs revision" or "not attempted", aiReasoning: "reasoning for specific evaluation"}. generated by iterating through the evaluationCriteria.
     TEXT
     structure_without_skills = <<~TEXT
       Please review the student's work. Respond in correctly formatted JSON.
       evaluationCriteria should be a copy of #{evaluation_criteria}.
-      aiEvaluation should be your overall assessment of the student's work. Respond with "great", "ok", or "needs revision".
+      aiEvaluation should be your overall assessment of the student's work. Respond with "Great", "Ok", or "Needs revision".
```

Comment:
We could consider putting these in a constant, maybe too early for that? And if we want to do any analysis with past and future data combined we'll need to remember to match the casing.